docs/code/MlNetCookBook.md

```diff
@@ -244,6 +245,35 @@ var reader = mlContext.Data.TextReader(new[] {
 var data = reader.Read(dataPath);
 ```
 
+## How do I debug my experiment?
+
+Most ML.NET operations are 'lazy': they are not actually processing data, they just validate that the operation is possible, and then defer execution until the output data is actually requested. This provides good efficiency, but makes it hard to step through and debug the experiment.
+
+In order to improve debug-ability, we have added a `Preview()` extension method to all data views, transformers, estimators and readers:
+
+- `Preview` of a data view contains first 100 rows (configurable) of the data view, encoded as objects, in a single in-memory structure.
+- `Preview` of a transformer takes data as input, and outputs the preview of the transformed data.
+- `Preview` of an estimator also takes data as input, fits an 'approximated model' on the first 100 rows (configurabele) of data, and then outputs the preview of the resulting transformer.
+
+This feature is only available starting from version 0.8.
```

Comment:
This will become a true though out-of-date remark quickly (as I don't expect anyone to stay on <0.8 for long). 

Perhaps we should file an issue to remove this in a month?